modules/lang-painless/src/main/java/org/opensearch/painless/antlr/PainlessParser.java

```diff
@@ -1,35 +1,4 @@
-/*
```

Comment:
Looks like I accidentally removed these when re-generating parser config. Will add back once actions finish running.

Reply:
Added back in https://github.com/opensearch-project/OpenSearch/pull/11816/commits/fdfb6407e386029f636c5df68b68f41c2b60272e